gapic/src/main/com/google/gapid/models/Devices.java

```diff
@@ -246,6 +247,22 @@ public void removeListener(Listener listener) {
     listeners.removeListener(listener);
   }
 
+  public static String getVulkanDriverVersions(Device.Instance dev) {
+    StringBuilder version = new StringBuilder("N/A");
+    VulkanDriver vkDriver = dev.getConfiguration().getDrivers().getVulkan();
+    Boolean first = true;
```

Comment:
nit: can use `boolean` instead in the future (in Java, primitive types started with a capital letter (`Boolean`) are boxed type, they would have some affiliated helper functions, if in no need of those functions, then the non-capital types (`boolean`) would be enough.

Reply:
Good catch, I've updated to a simple `boolean`, thanks!